tasmota/language/en_GB.h

```diff
@@ -681,6 +681,7 @@
 #define D_SENSOR_WINDMETER_SPEED "WindMeter Spd"
 #define D_SENSOR_TELEINFO_RX   "TInfo Rx"
 #define D_SENSOR_TELEINFO_ENABLE "TInfo EN"
+#define D_SENSOR_LMT01_PULSE   "LMT01 Pulse"
```

Comment:
Hi,

This line is missing in the rest of the language files. Please add it to them. Thanks.

Reply:
Oops, sorry.  Done now!